src/benchmark/file_io_read_micro_benchmark.hpp

```diff
@@ -46,5 +46,10 @@ class FileIOMicroReadBenchmarkFixture : public MicroBenchmarkBasicFixture {
   void pread_atomic_single_threaded(benchmark::State& state);
   void pread_atomic_random_multi_threaded(benchmark::State& state, uint16_t thread_count);
   void pread_atomic_random_single_threaded(benchmark::State& state);
+  void mmap_read_single_threaded(benchmark::State& state, int mmap_mode_flag, int access_order);
+  void mmap_read_multi_threaded(benchmark::State& state, int mmap_mode_flag, uint16_t thread_count, int access_order);
```

Comment:
The thread count could be a uint8_t.
Even a char would be sufficient i guess.

Reply:
For the sake of consistency I'd let it stay that way